utils/swaggerDefinition.js

```diff
@@ -140,6 +140,99 @@ const swaggerOptions = {
             }
           }
         },
+        contributions: {
+          type: 'object',
+          properties: {
+            noteworthy: {
+              type: 'array',
```

Comment:
Object not added correctly.
I am not getting correct object in swagger docs.
<img width="486" alt="image" src="https://user-images.githubusercontent.com/22790328/105934549-6e6d5b00-6076-11eb-988d-683ce2126898.png">

Reply:
Refer: https://swagger.io/docs/specification/data-models/data-types/#array